webapp/src/components/Conference/ParticipantList/ParticipantList.scss

```diff
@@ -20,14 +20,33 @@
     margin: 0;
     padding: 8px;
     overflow: auto;
+    flex-grow: 1;
   
     li {
       list-style: none;
       color: #0a2136;
-      padding-left: 8px;
+      padding: 0 8px;
       height: 32px;
       line-height: 32px;
       cursor: default;
+      display: flex;
+      border-radius: 8px;
+
+      .ParticipantName {
+        flex-grow: 1;
+      }
+
+      .AdmitButton {
+        min-height: 2em;
+        padding:  0 0.8em;
+      }
+
+      .ParticipantStatus {
+        display: flex;
+        align-items: center;
+        gap: 1em;
```

Comment:
is it conscious decision to use `em` or would it be better to use `rem`?

Reply:
I always used `em`. Is it better to use `rem` instead?